seedvault-backup-parser.spec

```diff
@@ -0,0 +1,44 @@
+%define debug_package %{nil}
+
+%define mybuildnumber %{?build_number}%{?!build_number:1}
+
+Name:           seedvault-backup-parser
+Version:        2020.10.23
+Release:        %{mybuildnumber}%{?dist}
+Summary:        Parse Seedvault backups
+BuildArch:      noarch
+
+License:        GPLv2+
+URL:            https://github.com/Rudd-O/seedvault_backup_parser
```

Comment:
This is not the link of this repository, but your fork

Reply:
I will fix this.  Thanks for the feedback.